docs/xxlegacyxlogoption.md

```diff
@@ -24,14 +24,21 @@
 
 # -XX:[+|-]LegacyXlogOption
 
-Controls processing of the [`-Xlog`](xlog.md) option. When `-XX:-LegacyXlogOption` is set, which is the default, the `-Xlog` option is not recognized.
-When `-XX:+LegacyXlogOption` is set, the `-Xlog` option is equivalent to the [`-Xsyslog`](xsyslog.md) option. 
+Controls processing of the `-Xlog` option.
 
 ## Syntax
 
-| Setting                  | Effect     | Default                                                                            |
-|--------------------------|------------|:----------------------------------------------------------------------------------:|
-| `-XX:+LegacyXlogOption` | Process -Xlog  |                                                                              |
-| `-XX:-LegacyXlogOption` | Ignore -Xlog   | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span> |
+| Setting                 | Effect                                     | Default                                                                       |
+|-------------------------|--------------------------------------------|:-----------------------------------------------------------------------------:|
+| `-XX:+LegacyXlogOption` | Enable legacy `-Xlog` behavior             |                                                                               |
+| `-XX:-LegacyXlogOption` | Process `-Xlog` requests for GC logging    | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>|
 
-<!-- ==== END OF TOPIC ==== xxdisableexplicitgc.md ==== -->
\ No newline at end of file
+## Explanation
+
+From Eclipse OpenJ9 0.24.0, the `-Xlog` option is replaced by the [`-Xsyslog`](xsyslog.md) option. The `-XX:[+|-]LegacyXlogOption` controls how the `-Xlog` option is processed.
+
+- When `-XX:-LegacyXlogOption` is set, the `-Xlog` option is not recognized unless garbage collection (GC) options are specified (for example, `-Xlog:gc[:stderr|:file=<filename>]`). For more information, see [`-Xlog`](xlog.md).
```

Comment:
Not strictly true, `-Xlog` and `-Xlog:all` are recognized as forms that enabled gc logging.

Reply:
>Not strictly true, -Xlog and -Xlog:all are recognized as forms that enable gc logging.

Maybe(?):

"When `-XX:-LegacyXlogOption` is set, the `-Xlog` option is not recognized unless a form of this option is run that requests garbage collection (GC) logging (for example, `-Xlog:gc[:stderr|:file=<filename>]`)."

Reply:
Yes, this is better.